odoo/api.py

```diff
@@ -668,7 +668,7 @@ def companies(self):
         user_company_ids = self.user._get_company_ids()
```

Comment:
No need to be call in some cases. Can you move to the right location (like for `def company`) ?

Reply:
This is the entire point of the fix :sweat_smile: